lib/utils/log/slog_handler.go

```diff
@@ -222,52 +255,87 @@ func (s *SlogTextHandler) Handle(ctx context.Context, r slog.Record) error {
 	buf := newBuffer()
 	defer buf.Free()
 
-	if !r.Time.IsZero() {
+	if s.withTimestamp && !r.Time.IsZero() {
 		if s.cfg.ReplaceAttr != nil {
 			*buf = s.appendAttr(*buf, slog.Time(slog.TimeKey, r.Time))
 		} else {
 			writeTimeRFC3339(buf, r.Time)
 		}
 	}
 
-	var color int
-	var level string
-	switch r.Level {
-	case TraceLevel:
-		level = "TRACE"
-		color = gray
-	case slog.LevelDebug:
-		level = "DEBUG"
-		color = gray
-	case slog.LevelInfo:
-		level = "INFO"
-		color = blue
-	case slog.LevelWarn:
-		level = "WARN"
-		color = yellow
-	case slog.LevelError:
-		level = "ERROR"
-		color = red
-	case slog.LevelError + 1:
-		level = "FATAL"
-		color = red
-	default:
-		color = blue
-		level = r.Level.String()
-	}
+	// Processing fields in this manner allows users to
+	// configure the level and component position in the output.
+	// This matches the behavior of the original logrus. All other
+	// fields location in the output message are static.
+	for _, field := range s.cfg.ConfiguredFields {
+		switch field {
+		case levelField:
+			var color int
+			var level string
+			switch r.Level {
+			case TraceLevel:
+				level = "TRACE"
+				color = gray
+			case slog.LevelDebug:
+				level = "DEBUG"
+				color = gray
+			case slog.LevelInfo:
+				level = "INFO"
+				color = blue
+			case slog.LevelWarn:
+				level = "WARN"
+				color = yellow
+			case slog.LevelError:
+				level = "ERROR"
+				color = red
+			case slog.LevelError + 1:
+				level = "FATAL"
+				color = red
+			default:
+				color = blue
+				level = r.Level.String()
+			}
 
-	if !s.cfg.EnableColors {
-		color = noColor
-	}
+			if !s.cfg.EnableColors {
+				color = noColor
+			}
 
-	level = padMax(level, trace.DefaultLevelPadding)
-	if color == noColor {
-		*buf = s.appendAttr(*buf, slog.String(slog.LevelKey, level))
-	} else {
-		*buf = fmt.Appendf(*buf, " \u001B[%dm%s\u001B[0m", color, level)
-	}
+			level = padMax(level, trace.DefaultLevelPadding)
+			if color == noColor {
+				*buf = s.appendAttr(*buf, slog.String(slog.LevelKey, level))
+			} else {
+				*buf = fmt.Appendf(*buf, " \u001B[%dm%s\u001B[0m", color, level)
+			}
+		case componentField:
+			component := s.component
+			// If there is no component for the handler check if one was provided with
```

Comment:
```suggestion
			// If there is no component for the handler check, create one with the attribute value of the log message.
```
maybe that? it's not clear otherwise.